cvat-ui/src/components/annotation-page/standard3D-workspace/styles.scss

```diff
@@ -263,3 +262,44 @@
     text-align: center;
     vertical-align: middle;
 }
+
+.react-resizable {
+    position: relative;
+}
+
+.react-resizable-handle-horizontal {
+    position: absolute;
+    margin-left: auto;
+    width: 100%;
+    margin-right: auto;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    background-color: grey;
+    height: $grid-unit-size/2;
+    cursor: ns-resize;
+}
+
+.react-resizable-handle-vertical-side {
+    position: absolute;
+    width: $grid-unit-size/2;
+    margin-right: auto;
+    top: $grid-unit-size * 4.5;
+    right: 0;
+    bottom: 0;
+    background-color: grey;
+    height: 100%;
+    cursor: ew-resize;
+}
+
+.react-resizable-handle-vertical-top {
+    position: absolute;
+    width: $grid-unit-size/2;
+    margin-right: auto;
+    top: $grid-unit-size * 4.5;
+    right: 0;
+    bottom: 0;
+    background-color: grey;
+    height: 100%;
+    cursor: ew-resize;
+}
```

Comment:
For classes, defined in CVAT, please, use ``cvat-`` prefix.

Reply:
Resolved